pylinalg/func/misc.py

```diff
@@ -54,4 +67,14 @@ def aabb_transform(aabb, matrix, /, *, out=None, dtype=None):
     alignment axis stay the same.
 
     """
-    raise NotImplementedError()
+
+    aabb = np.asarray(aabb, dtype=float)
+    matrix = np.asarray(matrix, dtype=float).transpose((-1, -2))
+
+    if out is None:
+        out = np.empty_like(aabb, dtype=dtype)
+
+    out[:] = (vector_make_homogeneous(aabb) @ matrix)[..., :-1]
+    out.sort(axis=-2)
```

Comment:
I don't think this is correct. You need to evaluate the transform on all 8 corners of the aabb to find the new extrema.

Reply:
Yep, that's right. Not sure what I was thinking there. I'll update the code.

Reply:
Done. @Korijn please check if you are okay with the heavy usage of `as_strided` here. I wanted to keep the ability to do batched `aabb_transform`, so I am being "clever" on how to build the array of corner points.

The result is efficient, but a bit verbose, so you need to decide if we want that here, or if we prefer something more readable but perhaps less efficient.